source/DSCResources/MSFT_SccmIniFIle/MSFT_SccmIniFIle.psm1

```diff
@@ -0,0 +1,1107 @@
+$script:configMgrResourcehelper = Join-Path -Path $PSScriptRoot -ChildPath '..\..\Modules\ConfigMgrCBDsc.ResourceHelper'
+
+Import-Module -Name $script:configMgrResourcehelper
+
+<#
+    .SYNOPSIS
```

Comment:
you have some long lines in the Synopsis consider starting a new line on a few of them.

Reply:
Fixed. Will be in the next push.